src/metric_model_reporter.cc

```diff
@@ -230,13 +242,16 @@ MetricModelReporter::InitializeSummaries(
         &Metrics::FamilyInferenceComputeInferSummary();
     summary_families_["compute_output_duration"] =
         &Metrics::FamilyInferenceComputeOutputSummary();
-    // Cache: Note that counts are included in summary
-    summary_families_["cache_hit_duration"] = &Metrics::FamilyCacheHitSummary();
-    summary_families_["cache_miss_duration"] =
-        &Metrics::FamilyCacheMissSummary();
+    // Only create cache metrics if cache is enabled to reduce metric output
+    if (config_.cache_enabled_) {
```

Comment:
Should we include the other cache metrics in the summary? I think this might be confusing. We can just have summary not work with caching enabled as before.